boards/sltb001a/include/periph_conf.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2015-2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sltb001a
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for the SLTB001A starter kit
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "cpu.h"
+
+#include "periph_cpu.h"
+
+#include "em_cmu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ * @{
+ */
+#ifndef CLOCK_HF
```

Comment:
could we make this block more user-friendly, by simply defining the target clock speed (as in `#define CLOCK_CORECLOCK  (48000000U)` or similar), and deduct the clock setting from this? Or is the CPU running on a fixed clock speed anyway?

Reply:
While I do not object, I would like to leave it as-is for now. It would only replace `CLOCK_CORE_DIV`, since you still need to be able to select a different clock source (also check [this](https://github.com/basilfx/EFM2RIOT/blob/master/dist/doc/SLTB001A.md#clock-selection)).

Reply:
agreed, this might as well be 'nicified' in the future.